src/App.tsx

```diff
@@ -1,11 +1,53 @@
 import './App.css';
 import { Button } from './ui/button/button';
+import { Title } from './ui/title/title';
+import { Bar } from './features/header/header';
+import { Tabs } from './ui/tabs/tab';
+import { BigPostCard } from './ui/post_card/big_post_card/big_post_card';
```

Comment:
file names & directory names should have kebab-cased form
`big_post_card` -> `big-post-card`